### PR TITLE
5004, 5584 & 3634 - settings page design, hide nav on login and remove profile page

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/pages/Login/Login.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ export default function Login ({ history: { push } }) {
     headerProps={{
       title: 'HoloPort'
     }}
+    showSideMenu={false}
```
Comment (JettTech) [inline src/pages/Login/Login.js:46]: Yea! :clap:

File: src/pages/Settings/Settings.js [hunk 2]
```diff
@@ -28,147 +29,113 @@ function useUpdateVersion () {
...
   const updateVersion = useUpdateVersion()
 
   const availableVersion = get('versionInfo.availableVersion', status)
   const currentVersion = get('versionInfo.currentVersion', status)
   const updateAvailable = (!isEmpty(availableVersion) && !isEmpty(currentVersion) && (availableVersion !== currentVersion))
 
+  const title = (settings.hostName ? `${settings.hostName}'s` : 'Your') + ' HoloPort'
+
+  const ports = (() => {
+    const portsObject = (omit('__typename', get('ports', status)) || {})
```
Comment (JettTech) [inline src/pages/Settings/Settings.js:50]: Nice update.

**Review by JettTech — APPROVED** [review]
Some clean updates.  I'd like to review a few changes in person, but we can merge it now and review it synchronously soon after.  (Also, just to verify, this implementation of the HPOS API has not yet been matched with the most recently published HPOS API, correct?)

File: src/pages/Settings/Settings.js [hunk 3]
```diff
@@ -28,147 +29,113 @@ function useUpdateVersion () {
...
 }
 
-export function UpdateSoftwareModal ({ settings, handleClose, toggleModal, availableVersion, updateVersion }) {
+export function VersionUpdateModal ({ settings, handleClose, modalVisible, availableVersion, updateVersion }) {
   if (!availableVersion) return null
   const onYes = () => {
     updateVersion()
     handleClose()
   }
   return <Modal
```
Comment (JettTech) [inline src/pages/Settings/Settings.js:136]: One style note.  The Yes and No buttons in the modal, still have the original link styling  (ie: blue with underline).  Please just update this one style. :+1:

**Review by JettTech — APPROVED** [review]
I'll keep this as approved.. but please just double check the styling for the modal buttons as well as the ssh button in Settings.

**Comment by robbiecarlton** [comment]
@JettTech 

SSH button I made a judgement call that it's close enough. Right now using an out of the box component which is convenient. Will run that one past Jamison + Zoe.

I moved the modal out of scope because C and I couldn't find any designs for it, so there is a new ticket for design and implementation of that modal (along with a couple of other things).

**Comment by robbiecarlton** [comment]
Re HPOS API: correct, that's out of scope for this ticket as this is a design implementation ticket. I think there's already a ticket for that in the backlog.

**Pull request merged**